client/app/states/login/login.state.js

```diff
@@ -36,6 +36,10 @@
       password: API_PASSWORD,
     };
 
+    if ($window.location.href.includes("?timeout")) {
+      Notifications.message('danger', '', __('Your session has timed out.'), true);
```

Comment:
@AllenBW Could `Notifications` and `EventNotifications` be two separate things? Reason I ask is because `EventNotifications` probably expects the authentication token to be already present.
When I replace all `Notifications` calls with `EventNotifications` in that file, I get this at login 
<img width="1390" alt="screen shot 2016-12-15 at 9 17 01 am" src="https://cloud.githubusercontent.com/assets/1538216/21234432/78e5e960-c2a7-11e6-94e4-1b76dbebbdf8.png">